src/components/nodes/startEvent/startEventConfig.js

```diff
@@ -0,0 +1 @@
+export const startEventDiameter = 36;
```

Comment:
What's the advantage of having this single constant in a file?

Reply:
I've just seen them used in tests, which makes sense. I still wonder about not extracting them to separate files though.

Reply:
With this, and the task height/width, if I keep them in the same `index.js` file, Cypress complains that it can't parse the file due to the imports:

```bash
SyntaxError: 'import' and 'export' may appear only with 'sourceType: module'
    at EventEmitter.handler (/Users/robertkomaromi/Library/Caches/Cypress/3.4.1/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/util.js:67:29)
    at emitOne (events.js:116:13)
    at EventEmitter.emit (events.js:211:7)
    at ChildProcess.<anonymous> (/Users/robertkomaromi/Library/Caches/Cypress/3.4.1/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/util.js:25:29)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at emit (internal/child_process.js:772:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

Our babel setup should handle the ES module synatax, so it appears that Cypress doesn't run the test files through babel, or there's some configuration mismatch. I bug into it a little but didn't get far so decided this was the easier option.

I can look into it more, as it would be nicer so keep them in the same file.

Reply:
Interesting. Let's not hold this up for now but we can fix that later if you like.

Reply:
Sounds good; I can fix it later and not let this hold it up right now.